src/pages/vendor/VendorAddProductPage/ProductDescriptionForm/index.tsx

```diff
@@ -323,12 +341,27 @@ function ProductDescriptionForm() {
                 const selectedOptions = selected as colorOptions[];
 
                 if (selectedOptions.length === 0) {
-                  return <Typography>{colors[0].value}</Typography>;
+                  return (
```

Comment:
you can write if(!selectedOptions.length)

Reply:
fixed